Cargo.toml

```diff
@@ -4,14 +4,14 @@ version = "0.1.5"
 authors = ["NikVolf <nikvolf@gmail.com>"]
 
 [dependencies]
-tokio-uds = "0.1"
-tokio-named-pipes = { git = "https://github.com/nikvolf/tokio-named-pipes", branch = "stable" }
+tokio-uds = "0.2"
+tokio-named-pipes = { version = "0.2", git = "https://github.com/poanetwork/tokio-named-pipes" }
```

Comment:
should not change

Reply:
Yes, as I mentioned, I'll change back as soon as changes are merged :)